reference/lemmy.v1.yaml

```diff
@@ -0,0 +1,668 @@
+openapi: 3.0.0
+info:
+  title: lemmy
+  version: '1.0'
+servers:
+  - url: 'http://localhost:3000'
```

Comment:
You could also use one of our test servers (or multiple):

- https://voyager.lemmy.ml/
- https://enterprise.lemmy.ml/
- https://ds9.lemmy.ml/